test/hbbft_worker.erl

```diff
@@ -67,37 +69,60 @@ verify_chain([A, B|_]=Chain, PubKey) ->
             false
     end.
 
+verify_block_fit([A, B | _], PubKey) ->
```

Comment:
It would be good to make verify_chain's second clause use verify_block_fit (and if verify_chain returns true, continue to the next block). That would reduce quite a lot of duplicate code.

Reply:
updated in 69fd63b